transforms/acceptance/__testfixtures__/tracking_page_events.output.js

```diff
@@ -1,14 +1,10 @@
 import { setupApplicationTest } from 'ember-qunit';
+import { assertTrackEvent, setupIntercomEventService, assertTrackAnalyticsEvent } from "embercom/tests/helpers/services/intercom-event";
 import { visit } from '@ember/test-helpers';
-import {
-  assertTrackEvent,
-  assertTrackAnalyticsEvent,
-  setupIntercomEventService,
-} from 'embercom/tests/helpers/services/intercom-event';
 
 module('some module', function(hooks) {
-  setupApplicationTest(hooks);
   setupIntercomEventService(hooks);
+  setupApplicationTest(hooks);
```

Comment:
Should `setupApplicationTest` not be called before everything else?

It's a bit hacky but what we do elsewhere is we find `setupApplicationTest` and replace it with `replaceWith(setupApplicationTest(hooks);\nsetupIntercomEventService(hooks);`

Reply:
Oh is it necessary to have `setupApplicationTest` at first? I didn't know that. So shall we assume then `setupApplicationTest` is always available?

Reply:
Yeah it's always there (the community codemod adds it). I'm just concious because it does setup (e.g. sets up the store) that is used by some helpers

Reply:
Alright let me make the change then.

Reply:
Got it. Let me update the PR.

Reply:
Updated it by doing a check if `setupApplicationTest` exists then append it and if not add both to the file. Updated the test files also